### ISSUE TITLE
Question Démo Pré-Soutenance

### ISSUE THREAD

**Issue by pamellahani** [2024-2025-ECOM-INFO5-G03/App#9]
Bonjour, 

Est ce que, pour la pré-soutenance du 15/10, faut-il avoir une version minimale avec setup de la base de données déployée sur la VM Azure, (aka faire une demo de notre code déployé sur Azure) ?

Merci.

**Comment by donsez** [comment]
Oui. une application générée sans modification au minimun. Vous pouvez customiser le style.
